### PR TITLE
Don't show the upgrade notice for beta versions.

### PR DESCRIPTION
**Pull request by robindevitt**
Related issue - #453 

Currently when on a beta version of Altis, the upgrade notice is shown. 

This PR ensures that when your "current" version of Altis, is equal to or greater than the "latest" version you won't be shown the upgrade notice.

### CODE REVIEW

File: inc/upgrades/namespace.php
Comment (rmccue) [inline inc/upgrades/namespace.php:358]: I think we should fix the check inside `is_version_latest()` instead

Comment (robindevitt) [reply inc/upgrades/namespace.php]: Thanks for the suggestion Ryan, definitely a better approach!

File: inc/upgrades/namespace.php
Comment (robindevitt) [inline inc/upgrades/namespace.php:132]: Would this be preferred?
`return ( isset( $releases[0]['version'] ) && $version >= $releases[0]['version'] ?? false );`

Comment (rmccue) [reply inc/upgrades/namespace.php:132]: `??` is the [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) which replaces the need to use isset.

That said, the intention here is to handle if there's no value available to force the check to return true, so your code needs to be
```suggestion
	return $version >= ( $releases[0]['version'] ?? $version );
```

**Review by rmccue — CHANGES_REQUESTED** [review]

**Comment by rmccue** [comment]
I can't review this seeing as I wrote the suggestion :D

**Review by kovshenin — APPROVED** [review]
Looks good!

**Comment by robindevitt** [comment]
@rmccue I can't merge this until you approve the requested changes too, please. I tried to remove you but was only able to re-request a review. @kovshenin has approved the change too, your approval should allow the merge.

**Comment by kovshenin** [comment]
@robindevitt you're good to go, I removed the blocker 😆

**Comment by robindevitt** [comment]
@kovshenin is this worth me backporting to v13 & v12? Maybe include v11?

**Comment by kovshenin** [comment]
This only affects future betas. The first v13 beta has already shipped, and we're not planning a second one currently, so I don't think it needs backporting.

**Pull request merged**
